src/Pages/Playground/components/playgrounApiDocs.tsx

```diff
@@ -0,0 +1,587 @@
+                {item}
+              </span>
+            )}
+          </For>
+          <Show when={min !== undefined || max !== undefined}>
+            <span class="type-keyword text-[#c2c2c2] text-[14px] mt-2 mr-2">
+              [{min};{max? max : '∞'}]
+            </span>
+          </Show>
+          <Show when={defaultValue}>
+            <span class="type-keyword default text-[#c2c2c2] text-[14px] mt-2 mr-2">
+              default:
+            </span>
+            <code class="type-default text-[#2f4f4f] text-[16px] border border-solid rounded border-[#eaeced] py-0.5 px-1 font-bold">
+              {defaultValue}
+            </code>
+          </Show>
+          <Show when={examples}>
+            <span class="type-keyword default text-[#c2c2c2] text-[14px] mt-2 mr-2">
+              Example:
+            </span>
+            <code class="type-default text-[#ffffff] text-[16px] border border-solid rounded border-[#eaeced] py-0.5 px-1 font-bold">
+              {examples}
+            </code>
+          </Show>
+        </div>
+      </div>
+      <Show when={description}>
+        <div class="signature-description  text-[#c2c2c2] float-left max-w-[600px] w-full leading-[17px]">
+          <p
+            class="m-0 mt-2 leading-[2] text-[14px]"
+            innerHTML={strDescription}
+          ></p>
+        </div>
+      </Show>
+      <Show when={items && showSignature()}>
+        <div class="signature-box-container mt-2 clear-both">
+          <div class="sub-box-container bg-[rgba(37,37,37,.4)">
+            <Box content={items} activeClass={activeClass()} />
+          </div>
+        </div>
+      </Show>
+      <Show when={!items && type === "object" && showSignature()}>
+        <div class="signature-box-container mt-2 clear-both">
+          <div class="sub-box-container bg-[rgba(37,37,37,.4)">
+              <Box content={property}  nested={true} />              
+          </div>
+        </div>
+      </Show>
+    </div>
+  );
+};
+
+const JsonSchema: Component<JsonSchemaType> = ({ json, mouseHoverFunc }) => {
+  return (
+    <div
+      class="source block px-6 py-7 bg-[#0e0e0e] mb-4"
+      onMouseEnter={() => mouseHoverFunc(1)}
+      onMouseLeave={() => mouseHoverFunc(0)}
+    >
+      <pre class="json-schema text-[#ffcd44] text-[14px] leading-[20px] max-w-full overflow-auto whitespace-pre-wrap">
+        {JSON.stringify(json, null, 2)}
+      </pre>
+    </div>
+  );
+};
+
+export const PlaygroundApiDocs: Component<PlaygroundApiDocsType> = ({
+  name,
+}) => {
+  // const [requestSchema, setRequestSchema] = createSignal({});
+  // const [responseSchema, setResponseSchema] = createSignal({});
+
+  // createEffect(() => {
+  //   // setSchemaName(name)
+  //   schemaSend(name).then(res => { setResponseSchema(res)})
+  //   console.log("request",responseSchema())
+  // })
```

Comment:
commented code

Reply:
This is a function for dynamic import of schemas from schema files of project based on their names, I commented it for further uses.

Reply:
I see, alright